### PR TITLE
Portfolio: Add desktop design

### PR DESCRIPTION
**Pull request by mateo951**
- Added media queries for desktop responsiveness in the header section
- Added media queries for desktop responsiveness in the header projects section
- Added media queries for desktop responsiveness in the aboutme section
  - Added fixes to media queries classes to fix containers issues with some sizes (768px - 1024px)
- Added media queries for desktop responsiveness in the form section
- Added linters corrections

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -411,3 +405,202 @@ section ul {
 #msg {
   min-height: 114px;
 }
+
+@media screen and (min-width: 768px) {
```
Comment (arslanbisharat) [inline style.css:409]: - [ ] The value for min-width property should be in range of 992px - 1100px

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested , 

Hi @mateo951, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

_This is an additional review as a result of the second opinion request._

The project was for desktop versions, so in our guidelines (the ones CRs check to make reviews) we added a line about using 992px-1100px as a minimum width since this is the right range for desktop/laptop screens (768px is for tablets), which explains why the CR asked you for this

However, I do agree that this should be more open, and you should be able to make it work even for tablets, as long as it works fine for desktops, therefore, we're changing the guidelines to make it more flexible for Micronauts to make their project work for different screens, **as long as it's working on desktops**, thank you for bringing this to our attention!

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**